resources/views/pages/Public/Join.svelte

```diff
@@ -8,98 +8,135 @@
 </script>
 
 <!-- Adapted from https://v1.tailwindcss.com/components/forms#form-grid -->
-<form class="h-screen-no-nav border border-gray-200 border-gray-700 bg-gray-800 p-3 shadow-md">
-    <div class="-mx-3 mb-6 flex flex-wrap">
-        <div class="mb-6 w-full px-3 md:mb-0 md:w-1/2">
-            <label class="mb-2 block text-xs font-bold uppercase tracking-wide text-gray-200" for="grid-name">
-                Your name
-            </label>
-            <input
-                class="mb-3 block w-full appearance-none rounded border border-red-500 bg-gray-200 py-3 px-4 leading-tight text-gray-700 focus:bg-white focus:outline-none"
-                id="grid-name"
-                type="text"
-            />
-            <p class="text-xs italic text-red-500">Please fill out this field.</p>
-        </div>
-        <div class="w-full px-3 md:w-1/2">
-            <label class="mb-2 block text-xs font-bold uppercase tracking-wide text-gray-200" for="grid-age">
-                Your age
-            </label>
-            <input
-                class="block w-full appearance-none rounded border border-gray-200 bg-gray-200 py-3 px-4 leading-tight text-gray-700 focus:border-gray-500 focus:bg-white focus:outline-none"
-                id="grid-age"
-                type="text"
-            />
-        </div>
+<form class="h-screen-no-nav border border-gray-700 bg-gray-800 p-3 shadow-md" id="join-form">
+  <div class="text-gray-200 text-center">
+    <h1 class="font-bold lg:text-4xl">JOIN ARCOMM</h1>
+    <p class="mt-6">
+      We're looking for well adjusted people who listen to and respect others. While having experience in our games is welcome, it's in no way required. Aspiring members should make an effort to attend the weekly operations, 
+      be accountable for their actions, and generally have a good sense of humor.
+    </p>
+  </div>
+  <hr class="h-px my-8 bg-gray-200">
+  <div class="-mx-3 mb-6 flex flex-wrap">
+      <div class="mb-6 w-full px-3 md:mb-0 md:w-1/2">
+          <label class="mb-2 block text-xs font-bold uppercase tracking-wide text-gray-200" for="grid-name">
```

Comment:
There's a lot of repeated styling used for each input, we should consider how it can be simplified

Reply:
I think I'd need time to work on this sort of thing, it's whether this is a now thing or something I revisit post main page completion?